packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -16,8 +16,13 @@
         id="files-breadcrumb"
         data-testid="files-breadcrumbs"
         class="oc-p-s"
+        :show-contextmenu="true"
```

Comment:
is this needed? Can we not just check for the slot.contextMenu in ODS?

Reply:
Good idea, I'll check (and update) the ODS